demo_nodes_cpp_native/package.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
+  <name>demo_nodes_cpp_native</name>
+  <version>0.0.2</version>
+  <description>
+    C++ nodes which access the native handles of the rmw implemenation.
+  </description>
+  <maintainer email="dthomas@osrfoundation.org">Dirk Thomas</maintainer>
+  <license>Apache License 2.0</license>
+
+  <buildtool_depend>ament_cmake</buildtool_depend>
+
+  <build_depend>rclcpp</build_depend>
+  <build_depend>rmw_implementation_cmake</build_depend>
```

Comment:
What is this dependency used for (I didn't see it being used in the cmake code)?